analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Sonar/PublicMethodArgumentsShouldBeCheckedForNull.CSharp11.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Program
```

Comment:
Maybe it's worth adding a test case also for static virtual method in interface?